docs/proxies.md

```diff
@@ -314,23 +314,28 @@ flowchart
 
     <em><a href='https://github.com/NHSDigital/communications-manager-api/blob/release/proxies/shared/policies/RaiseFault.403ServiceBan.xml'>RaiseFault.403ServiceBan</a></em>"]
     403.1 --> E
-    Q6.1 --> |No| CN["<a href='https://github.com/NHSDigital/communications-manager-api/blob/release/docs/proxies.md#content-negotiation'>Content Negotiation</a>"]
+    Q6.1 --> |No| Q7{Prefer 425 error?}
+    Q7 --> |Yes| 425["Raise 425 Retry Too Early
+
+    <em><a href='https://github.com/NHSDigital/communications-manager-api/blob/release/proxies/shared/policies/RaiseFault.425RetryTooEarly.xml'>RaiseFault.425RetryTooEarly</a></em>"]
+    425 --> E
+    Q7 --> |No| CN["<a href='https://github.com/NHSDigital/communications-manager-api/blob/release/docs/proxies.md#content-negotiation'>Content Negotiation</a>"]
     Q1 --> |No| VAT["Verify access token
 
     <em><a href='https://github.com/NHSDigital/communications-manager-api/blob/release/proxies/shared/policies/OauthV2.VerifyAccessToken.xml'>OauthV2.VerifyAccessToken</a></em>"]
-    VAT --> Q7{Is client credentials?}
-    Q7 --> |No| 403
-    Q7 --> |Yes| CN
-    CN --> Q8{Is sandbox environment?}
-    Q8 --> |Yes| Q9{Prefer 429 error?}
-    Q9 --> |Yes| 429["Raise 429 error
+    VAT --> Q8{Is client credentials?}
+    Q8 --> |No| 403
+    Q8 --> |Yes| CN
+    CN --> Q9{Is sandbox environment?}
+    Q9 --> |Yes| Q10{Prefer 429 error?}
+    Q10 --> |Yes| 429["Raise 429 error
 
     <em><a href='https://github.com/NHSDigital/communications-manager-api/blob/release/proxies/shared/policies/RaiseFault.429TooManyRequests.xml'>RaiseFault.429TooManyRequests</a></em>"]
     429 --> E
-    Q9 --> |No| E
-    Q8 --> |No| Q10{Is request on a known resource?}
-    Q10 --> |Yes| E
-    Q10 --> |No| 404["Raise 404 error
+    Q10 --> |No| E
+    Q9 --> |No| Q11{Is request on a known resource?}
+    Q11 --> |Yes| E
+    Q11 --> |No| 404["Raise 404 error
 
     <em><a href='https://github.com/NHSDigital/communications-manager-api/blob/release/proxies/shared/policies/RaiseFault.404NotFound.xml'>RaiseFault.404NotFound</a></em>"]
     404 --> E
```

Comment:
I believe there is a missing bit on the fault rules handler where we rethrow the 425 from the backend?

Reply:
Good spot. I thought the Target PreFlow Request was busy!